### PR TITLE
Differentiate share links

### PR DESCRIPTION
**Pull request by Pytal**
Before | After
--- | ---
image | image

Differentiate between share links especially when using a screenreader

### CODE REVIEW

**Review by PVince81 — APPROVED** [review]
:+1:

too bad the user cannot give a name to those links

**Review by jancborchardt — COMMENTED** [review]
@Pytal the numbers only appear with the second link, right? So "Share link" will initially not have a number. Then I'm thinking it might be best to cut the " (1)" from the first one and only number from then on, just like we do for filenames. when copying files.

**Review by artonge — APPROVED** [review]

**Review by jancborchardt — APPROVED** [review]

File: apps/files_sharing/src/components/SharingEntryLink.vue [hunk 1]
```diff
@@ -530,6 +539,15 @@ export default {
 			return window.location.protocol + '//' + window.location.host + generateUrl('/s/') + this.share.token
 		},
 
+		/**
+		 * Tooltip message for actions button
+		 *
+		 * @return {string}
+		 */
+		actionsTooltip() {
+			return t('files_sharing', 'Actions for "{title}"', { title: this.title })
```
Comment (JuliaKirschenheuter) [inline apps/files_sharing/src/components/SharingEntryLink.vue:548]: should we add an `index` (if it is available) here too?

Comment (Pytal) [reply apps/files_sharing/src/components/SharingEntryLink.vue:548]: Yes, `index` is already included in the title

**Comment by Pytal** [comment]
Wording adjusted slightly

**Pull request merged**